package.json

```diff
@@ -107,6 +125,9 @@
     "rimraf": "^2.6.3",
     "semantic-release": "^16.0.1",
     "underscore.string": "~3.3.4",
+    "tslint": "5.17.0",
```

Comment:
https://github.com/palantir/tslint
`TSLint is deprecated.`
Maybe it's a good moment to get rid of TSLint and update eslint configuration?

Reply:
Planned for future work. This is just as-is migration from carbon-factory.